webapp/sources/rudder/rudder-rest/src/test/resources/api/api_groups.yml

```diff
@@ -177,10 +173,8 @@ response:
                   ]
                 },
                 "origval" : {
-                  "rudder" : {
```

Comment:
that error is not really relevant (the value is created by hand in `MockService` above), but I wanted it to look like the real property, so that if people look at the test, they see something good